compiler-rt/test/tysan/violation-pr45282.c

```diff
@@ -18,7 +18,7 @@ int main(void) {
 
   // CHECK:      TypeSanitizer: type-aliasing-violation on address
   // CHECK-NEXT: WRITE of size 8 at {{.+}} with type double accesses an existing object of type float
-  // CHECK-NEXT:   in main violation-pr45282.c:25
+  // CHECK-NEXT:   in main {{.*violation-pr45282.c:25.*}}
```

Comment:
Nit: IMO having `{{.*}}` to swallow extra path components is slightly preferable, as that avoids the filename being part of a regex. Almost certainly not ever going to be a problem, but it reduces what can go wrong. Does your environment also have trailing components after the line number? If there's any whitespace between '5' and the extra text then FileCheck will (99% confident) ignore those anyway.

Similar thoughts apply to all the other similar regexes.

Reply:
Checked and the trailing .* can indeed be left out, I'll change that

Compiler-rt tests tend to have the file name in the check lines because you want to make sure that the sanitizer correctly informs the user where the problem is.